arch/src/x86_64/mod.rs

```diff
@@ -338,6 +341,144 @@ impl CpuidPatch {
     }
 }
 
+pub struct CpuidFeatureEntry {
+    pub function: u32,
+    pub index: u32,
+    pub feature_reg: CpuidReg,
+}
+
+impl CpuidFeatureEntry {
+    fn predefined_feature_entry_list() -> Vec<CpuidFeatureEntry> {
+        vec![
+            // Leaf 0x1, ECX/EDX, feature bits
+            CpuidFeatureEntry {
+                function: 1,
+                index: 0,
+                feature_reg: CpuidReg::ECX,
+            },
+            CpuidFeatureEntry {
+                function: 1,
+                index: 0,
+                feature_reg: CpuidReg::EDX,
+            },
+            // Leaf 0x7, EBX/ECX/EDX, extended features
+            CpuidFeatureEntry {
+                function: 7,
+                index: 0,
+                feature_reg: CpuidReg::EBX,
+            },
+            CpuidFeatureEntry {
+                function: 7,
+                index: 0,
+                feature_reg: CpuidReg::ECX,
+            },
+            CpuidFeatureEntry {
+                function: 7,
+                index: 0,
+                feature_reg: CpuidReg::EDX,
+            },
+            // Leaf 0x7 subleaf 0x1, EBX/ECX/EDX, extended features
```

Comment:
Wikipedia makes me think this one also returns in EAX too.

Reply:
Did you decide that you didn't need to do EAX.

Reply:
I meant to say you were right, and I already added the EAX of leaf 0x7 to the checklist. :)

Reply:
Finally added `EAX` of leaf 0x7, subleaf 0x1, to the checked list (and removed EBX/ECX/EDX). Sorry about not reading your comment and the Wiki page clearly.